sqlglot/dialects/presto.py

```diff
@@ -385,6 +385,9 @@ class Generator(generator.Generator):
                 "ARRAY_JOIN", self.func("ARRAY_AGG", e.this), e.args.get("separator")
             ),
             exp.Hex: rename_func("TO_HEX"),
+            exp.LowerHex: lambda self, e: self.func(
+                "LOWER", self.func("TO_HEX", self.sql(e, "this"))
+            ),
```

Comment:
This is not a redundant. This will rename_func the HEX to TO_HEX.

https://github.com/tobymao/sqlglot/blob/9402b5d6dc87807da7124a0a2fa19cfd9fba02d2/sqlglot/generator.py#L1311-L1315

Reply:
Ah got it, thanks for pointing it out